docs/devTools/README.md

```diff
@@ -0,0 +1,62 @@
+# Dev tools
+
+Local development tools for deploying the [any.sender contracts](https://github.com/PISAresearch/contracts.any.sender) and hosting a mock any.sender API instance. We also have a live [ropsten API](../addresses.md) for when you're ready to take it to the next level :)
+
+## Deploying contracts locally
+
+To deploy the contracts import the `deploy` object from the `@any-sender/client` package.  The deploy object has an `contracts` function which accepts a provider and returns a promise of the contracts. Call this at some point in your tests before wrapping objects with `any.sender()`.
+
+```
+contracts(provider: Provider, options?: DeployOptions) => Promise<Contracts>
+```
+
+#### provider: JsonRpcProvider | AsyncSendable | string
+Can be an ethers `JsonRpcProvider`, `AsyncSendable` or a json rpc url as a `string`.
+
+#### options?: DeployOptions
+```
+{
+    funder?: string | number;
+    shardInterval?: number;
+    compensationPeriod?: number;
+    withdrawalPeriod?: number;
+    deployMetaTxContracts?: boolean;
+}
+```
+
+* `funder`: The address or index of the wallet that will be used to fund the any.sender deployment. Defaults to the last signer returned by `provider.listAccounts`.
+* `shardInterval`: The data registry shard interval. The number of blocks before a shard can be rotated. Defaults to 180000.
+* `compensationPeriod`: The adjudicator compensation period. The number of blocks any.sender is allowed to give out compensation. Defaults to 180000.
+* `withdrawalPeriod`: The minimum amount of time between a withdrawal request and a withdrawal in the lockable deposit contract. Defaults to 360000.
+* `deployMetaTxContracts`: Whether to deploy the metatransaction hub, and the proxy account deployer. Default is true.
+
+#### Returns
+```
+Promise<{
+    funder?: string | number;
+    shardInterval?: number;
+    compensationPeriod?: number;
+    withdrawalPeriod?: number;
+    deployMetaTxContracts?: boolean;
+}>
+```
+Returns a promise of an object containing the `relay`, `adjudicator`, `lockableDeposit`, `relayer` contracts and the `receiptSigner` signer.
```

Comment:
I'm thinking from the dev perspective. All they really need to care about is Relay, Relayer, and ReceiptSigner right?

e.g. to verify the signed receipt from the API? Although the client tooling so do that anyway? For example in the example code, we don't really talk about any of them.

Reply:
They may want to check how to get compensation. For that they would need the adjudicator.

Tbh, the devs often dont need any of these contracts - if you look in the example they arent used, they just need to be deployed.

Reply:
yeah that is sort of my point. if you want to run a mock API, for the most part you dont care about any of the contracts or parameters. you just care about deployContract() since that is a necessary pre-setup. so a lot of this feels like extra-details and then the core thing they want to read is at the bottom of the docs.